maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -407,6 +408,23 @@ class IOSDriver(
         return element == null
     }
 
+    private fun tapContinueIfVisible() {
```

Comment:
Nit: Can you call this method something specific to the keyboard?
Example: 'dismissKeyboardIntroduction()'

Reply:
Hmmm, that's true 💯

Reply:
Making this change 👍